source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -29,8 +29,42 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+- Most search functionality included in the core platform now supports Meilisearch as an alternative to Elasticsearch. For instances deployed using Tutor, only Meilisearch is supported, and Tutor will automatically provision Meilisearch and create the indexes during the upgrade.
+   - After upgrading, you need to run ``manage.py cms reindex_studio --experimental --incremental`` to populate the new Studio Search index with your existing content (courseware/libraries). Studio search results will be incomplete until this command has finished.
```

Comment:
A required upgrade has you run with an `experimental` flag??

Reply:
Ah, I guess you don't _need_ to run it. It should say "If you wish to use the new Content Libraries (beta) or Studio Course Search features, you'll need to run..."